src/Symfony/Component/EventDispatcher/ContainerAwareEventDispatcher.php

```diff
@@ -105,7 +105,7 @@ public function removeListener($eventName, $listener)
     public function hasListeners($eventName = null)
     {
         if (null === $eventName) {
-            return (bool) count($this->listenerIds) || (bool) count($this->listeners);
+            return $this->listenerIds || $this->listeners;
```

Comment:
I am not symfony person, and I can't decide, but LondonPHP meetups advertise to write clean code, which I think this PR aiming for